src/entity/user/UserMarketingCta.ts

```diff
@@ -17,9 +17,10 @@ export class UserMarketingCta {
   readAt: Date;
 
   @ManyToOne(() => MarketingCta, {
-    lazy: true,
+    eager: true,
```

Comment:
Why do we need this? It means relation will be loaded each time we query the table through `find`. Which should not be the case.

Reply:
Wasn't supposed to add eager to it

Reply:
Intended to use `relations: ['marketingCta'],` in the findOne but was testing to see what was the difference and forgot to revert it